db/carrier_icons.yml

```diff
@@ -2,11 +2,52 @@
   name: ups
   label: UPS
   group: other
+-
+  name: usps
+  label: USPS
+  group: other
+-
+  name: sendle
+  label: Sendle
+  group: other
+  
+-
+  name: poste-italiane
+  label: PosteItaliane
+  group: other
+-
+  name: mondial-relay
+  label: MondialRelay
+  group: other
 -
   name: fedex
-  label: FedEx
+  label: Fedex
+  group: other
+-
+  name: evri
+  label: Evri
+  group: other
+-
+  name: dpd
+  label: Dpd
   group: other
 -
   name: dhl
   label: DHL
   group: other
+-
+  name: correos
```

Comment:
Per Correos new image guidelines, can you only include the symbol?

https://www.correos.com/en/sala-prensa/#imagen-marca

Alternatively, we could just add a correos_symbol.svg variant?

![CleanShot 2024-02-02 at 12 43 53@2x](https://github.com/Checkout-Blocks/shipping_carrier_icons/assets/3484527/4670f3e1-5edd-4ade-9e57-1b415d6c82fa)